tests/scripts/github-action-helper.sh

```diff
@@ -30,7 +30,7 @@ function find_extra_block_dev() {
   boot_dev="$(sudo lsblk --noheading --list --output MOUNTPOINT,PKNAME | grep boot | awk '{print $2}')"
   echo "  == find_extra_block_dev(): boot_dev='$boot_dev'" >/dev/stderr # debug in case of future errors
   # --nodeps ignores partitions
-  extra_dev="$(sudo lsblk --noheading --list --nodeps --output KNAME | grep -v loop | grep -v "$boot_dev" | head -1)"
+  extra_dev="$(sudo lsblk --noheading --list --nodeps --output KNAME | egrep -v "($boot_dev|loop|nbd)" | head -1)"
```

Comment:
@BlaineEXE please confirm if this looks good to you

Reply:
Looks good. I think `egrep -v "$boot_dev|loop|nbd"` is equivalent, but that isn't too important in this context.